components/SubmissionCard.tsx

```diff
@@ -1,23 +1,26 @@
 import React from 'react'
 import Button from './Button'
+import UserInfoImage from './UserInfoImage'
+import { User } from '../@types/user'
+import '../scss/submissionCard.scss'
 
 type Props = {
   userId: string
```

Comment:
I know we did not do on other types but should we not use `userId: ID!` instead of `string`

Reply:
what is `ID` used for?

Reply:
Nothing, I mixed up Type and GraphQL^^